packages/1155-contracts/src/royalties/CreatorRoyaltiesControl.sol

```diff
@@ -33,8 +33,8 @@ abstract contract CreatorRoyaltiesControl is CreatorRoyaltiesStorageV1, SharedBa
     }
 
     function _updateRoyalties(uint256 tokenId, RoyaltyConfiguration memory configuration) internal {
```

Comment:
how about just letting this take in a royalty recipient address, and removing the concept of `RoyaltyConfiguration` ?